libraries/chain/proposal_evaluator.cpp

```diff
@@ -97,33 +97,41 @@ namespace golos { namespace chain {
         : evaluator_impl<proposal_create_evaluator>(db) {
     }
 
-    void proposal_create_evaluator::do_apply(const proposal_create_operation& o) { try {
-        ASSERT_REQ_HF(STEEMIT_HARDFORK_0_18__542, "Proposal transaction creating"); // remove after hf
+#define CHECK_ALREADY_EXIST(OP, N, X, Y) \
```

Comment:
The macro is too specific. Not all objects are in the database _db and not all objects have identifiers consisting of two fields. Moreover, the use of operation argument names does not always correspond to the identifier of the missing object. So, in an operation they can be called "owner" or "author", but the user can more clearly see that the account "name" does not have comment with permlink.